cmd/kube-proxy/app/server_linux.go

```diff
@@ -91,17 +91,18 @@ func (s *ProxyServer) platformCheckSupported(ctx context.Context) (ipv4Supported
 	if isIPTablesBased(s.Config.Mode) {
 		// Check for the iptables and ip6tables binaries.
 		var ipts map[v1.IPFamily]utiliptables.Interface
-		ipts, err = utiliptables.NewDualStack()
+		var errDS error
+		ipts, errDS = utiliptables.NewDualStack()
```

Comment:
Don't need the separate declarations any more:

```golang
		ipts, errDS := utiliptables.NewDualStack()
```

Reply:
thanks Dan, pushed another commit to fix that